### ISSUE TITLE
"@implementation Foo {" not recognized

### ISSUE THREAD

**Issue by kjk** [drkameleon/iswift#33]
A snippet code like this:
```
@implementation AdvancedWorkingDirectoryWindowController {
    // Advanced working dir sheet
    IBOutlet NSMatrix* _windowDirectoryType;
    IBOutlet NSTextField* _windowDirectory;
    IBOutlet NSMatrix* _tabDirectoryType;
    IBOutlet NSTextField* _tabDirectory;
    IBOutlet NSMatrix* _paneDirectoryType;
    IBOutlet NSTextField* _paneDirectory;
}
```

is not parsed. It complains about "unrecognized {"

Found by trying to convert https://github.com/gnachman/iTerm2/blob/master/sources/AdvancedWorkingDirectoryWindowController.m

**Comment by drkameleon** [comment]
Hmm... Thanks a lot for letting me know! I'll fix it.

Basically, it is because the class variables (the `IBOutlet`s) are within the `@implementation` and not the interface.

If you had put it in the `@interface` part, it would have worked:

![screen shot 2015-08-31 at 10 51 05 pm](https://cloud.githubusercontent.com/assets/1265028/9590019/77156608-5032-11e5-8fc6-864fa5dc77a0.png)

In any case, it just found its way into my to-fix list; most likely by the very next release - 1.5, it will be fixed too! So... just a little patience! ;-)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.5.

----

**Example:**

```objective-c
@implementation AdvancedWorkingDirectoryWindowController {
    // Advanced working dir sheet
    IBOutlet NSMatrix* _windowDirectoryType;
    IBOutlet NSTextField* _windowDirectory;
    IBOutlet NSMatrix* _tabDirectoryType;
    IBOutlet NSTextField* _tabDirectory;
    IBOutlet NSMatrix* _paneDirectoryType;
    IBOutlet NSTextField* _paneDirectory;
}

@end
```

**Output:**

```swift
class AdvancedWorkingDirectoryWindowController {

	@IBOutlet var _windowDirectoryType: NSMatrix?
	@IBOutlet var _windowDirectory: NSTextField?
	@IBOutlet var _tabDirectoryType: NSMatrix?
	@IBOutlet var _tabDirectory: NSTextField?
	@IBOutlet var _paneDirectoryType: NSMatrix?
	@IBOutlet var _paneDirectory: NSTextField?
	
}
```
